app/views/exam_templates/_template_division_pane.html.erb

```diff
@@ -3,15 +3,17 @@
   <table>
     <thead>
       <tr>
-        <th><%= TemplateDivision.human_attribute_name('label') %></th>
-        <th><%= TemplateDivision.human_attribute_name('start') %></th>
-        <th><%= TemplateDivision.human_attribute_name('end') %></th>
+        <th class='required'><%= TemplateDivision.human_attribute_name('label') %></th>
+        <th class='required'><%= TemplateDivision.human_attribute_name('start') %></th>
+        <th class='required'><%= TemplateDivision.human_attribute_name('end') %></th>
         <th><%= t(:delete) %></th>
       </tr>
     </thead>
     <tbody>
-    <%= render partial: 'exam_templates/template_division',
-               locals: { f: f, disabled: disabled } %>
+    <% unless exam_template.nil? %>
```

Comment:
Similar to what I noted previous, I don't think `exam_template` should ever be nil when this partial is rendered?